app/views/users/_users_list.html.haml

```diff
@@ -14,3 +14,9 @@
         %th= t('.expire_date')
     %tbody.panel-body
       = render partial: 'user', collection: users
+
+  = render partial: 'application/paginate_footer',
+           locals: { entities: @users,
+                     paginate_class: 'users_pagination',
+                     items_count: @items_count,
```

Comment:
Use local value `items_count` here (if you are passing this in as a local var from the controller).